lexicons/com/atproto/server/getUserInviteCodes.json

```diff
@@ -0,0 +1,41 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.server.getUserInviteCodes",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Get all invite codes for a given user",
+      "parameters": {
+        "type": "params",
+        "properties": {
+          "includeUsed": { "type": "boolean", "default": true }
+        }
+      },
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["codes"],
+          "properties": {
+            "codes": {
+              "type": "array",
+              "items": {
+                "type": "ref",
+                "ref": "#invite"
+              }
+            }
+          }
+        }
+      }
+    },
+    "invite": {
+      "type": "object",
+      "requred": ["code", "available", "uses"],
```

Comment:
you got some typos here (`requred`, `uses` vs `users`)

Reply:
yup yup fortunately i ran head first into those 😛